meilisearch/tests/settings/test_settings_attributes_for_faceting_meilisearch.py

```diff
@@ -0,0 +1,46 @@
+import json
+import meilisearch
+from meilisearch.tests import BASE_URL, MASTER_KEY
+
+class TestAttributesForFaceting:
+
+    """ TESTS: attributesForFaceting setting """
+
+    client = meilisearch.Client(BASE_URL, MASTER_KEY)
+    index = None
+    attributes_for_faceting = ['title']
+    dataset_file = None
+    dataset_json = None
+
+    def setup_class(self):
+        self.index = self.client.create_index(uid='indexUID')
+        self.dataset_file = open('./datasets/small_movies.json', 'r')
+        self.dataset_json = json.loads(self.dataset_file.read())
+        self.dataset_file.close()
+
+    def teardown_class(self):
+        self.index.delete()
+
+    def test_get_attributes_for_faceting(self):
+        """ Tests getting the attributes for faceting """
+        response = self.index.get_attributes_for_faceting()
+        assert isinstance(response, object)
+        assert response == []
+
+    def test_update_attributes_for_faceting(self):
+        """Tests updating the attributes for faceting"""
+        response = self.index.update_attributes_for_faceting(self.attributes_for_faceting)
+        self.index.wait_for_pending_update(response['updateId'])
+        get_attributes_new = self.index.get_attributes_for_faceting()
+        assert len(get_attributes_new) == len(self.attributes_for_faceting)
+        assert get_attributes_new[0] == self.attributes_for_faceting[0]
```

Comment:
When we tests a list of elements, MeiliSearch doesn't always return the elements in the same order you are expecting. So you can't count on get_attributes_new[0] being the same 'first' element that you sent (self.attributes_for_faceting[0]). You won't see that behavior here as it is only 1 element, but you may iterate on `self.attributes_for_faceting` and check that each one of them is present in `get_attributes_new`, independently of the order MeiliSearch sends them in.